.github/scripts/update_gdscript_docs.py

```
@@ -0,0 +1,742 @@
+
+    decl.status = DocStatus.COMPLIANT
+
+
+# -----------------------------------------------------------------------------
+# Formatting & Invariants
+# -----------------------------------------------------------------------------
+def format_doc_lines(doc: MemberDoc, decl: MemberDeclaration, indent: str) -> List[str]:
+    lines = [f"{indent}## {doc.summary}\n"]
+    if doc.description and doc.description.strip():
+        lines.append(f"{indent}##\n")
+        for l in doc.description.strip().splitlines():
```

Ruff (0.16.3)

[error] Ambiguous variable name: `l`

(E741)